README.MD

```diff
@@ -394,75 +394,78 @@ pushed to the host.
 
```

Comment:
the section above this diff reads:
```
#### artifact_workspace
This setting moves where the built artifact should be. This is particularly
helpful when you want to pick and choose items from the build versus just
sending the whole artifact to your host. The `~/project` directory is the
that is committed to an "Artifact Build". By default, this value is also
`~/project`, so that everything that was built during the "build" job is then
pushed to the host.
```

I think it should read:

```
#### artifact_workspace
This setting moves where the built artifact should be. This is particularly
helpful when you want to pick and choose items from the build versus just
sending the whole artifact to your host. The `~/project` directory is always 
committed to an "Artifact Build". Therefore, by changing this directory to
something like `/tmp/project`, you can use rsync to move parts built by
the previous build script into `~/project`. By default, this value of this setting 
is `~/project`, so that everything that was built during the "build" job is then
pushed to the host.
```

Reply:
This has been updated.